tests/unit/mazepa/test_id_generation.py

```diff
@@ -249,12 +253,52 @@ def _gen_id_calls(_) -> dict[str, str]:
 
 
 def test_persistence_across_sessions() -> None:
-    # Create two separate processes - spawn ensures a new PYTHONHASHSEED is used
-    ctx = multiprocessing.get_context("spawn")
-    with ctx.Pool(processes=2) as pool:
-        result = pool.map(_gen_id_calls, range(2))
-
-    assert result[0] == result[1]
+    # Create two separate processes with different PYTHONHASHSEED values
+    # to ensure ID generation is consistent across different hash seeds.
+    # This workaround is necessary due to fork being the default
+    # multiprocessing method, meaning we cannot use 'spawn' method to test.
+
+    project_root = os.path.dirname(os.path.dirname(os.path.dirname(os.path.dirname(__file__))))
```

Comment:
?!

Reply:
Seemed like a reasonable way to get the root location relative to the test file no matter where you run it from.

Reply:
test_id_generation.py -> mazepa -> unit -> tests -> zetta_utils. I guess I can do `zetta_utils.__path__` instead.